OpenSim/Simulation/StatesTrajectory.h

```diff
@@ -279,6 +280,31 @@ class OSIMSIMULATION_API StatesTrajectory {
     bool isCompatibleWith(const Model& model) const;
     /// @}
 
+    /** Export the continuous state variables to a data table, perhaps to write
+     * to a file and postprocess in MATLAB/Python/Excel. The names of the
+     * columns in the table will be the full names of the continuous state
+     * variables (e.g., `knee/flexion/angle`).
+     *
+     * You must provide a model that is compatible with this trajectory,
+     * since only the model knows the names of the state variables.
+     *
+     * By default, all continuous state variables are written to the table
+     * (one per column). If you only want some of them to be written to the
+     * table, use the `stateVars` argument to specify their full names
+     * (e.g., `knee/flexion/angle`).
+     *
+     * @code
+     * auto allStateVars = states.exportToTable(model);
+     * auto kneeStates = states.exportToTable(model, {"knee/flexion/value",
+     *                                                "knee/flexion/speed"});
+     * @endcode
+     *
+     * @throws IncompatibleModel Thrown if the Model fails the check
+     *      isCompatibleWith().
+     */
+    TimeSeriesTable exportToTable(const Model& model,
```

Comment:
If you are mentioning in the comments about writing to file, maybe give readers a pointer on where to look if they want to do so -- *See DataAdapter for details on writing to files.*.